packages/toolpad-app/package.json

```diff
@@ -165,5 +165,10 @@
   "typings": "./index.d.ts",
   "publishConfig": {
     "access": "public"
+  },
+  "engines": {
+    "npm": "please-use-yarn",
+    "node": ">=18",
```

Comment:
Only the `node` field is necessary, we don't care which package manager they use.